### PR TITLE
chore: renames e2e test folder

### PR DESCRIPTION
**Pull request by amagyar-iohk**
### Description: 
Renames atala/prism references for e2e test
https://input-output.atlassian.net/browse/ATL-6471

### Checklist: 
- [x] My PR follows the [contribution guidelines](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm/blob/main/CONTRIBUTING.md) of this project
- [x] My PR is free of third-party dependencies that don't comply with the [Allowlist](https://toc.hyperledger.org/governing-documents/allowed-third-party-license-policy.html#approved-licenses-for-allowlist)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked the PR title to follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)

### CODE REVIEW

**Review by mkbreuningIOHK — COMMENTED** [review]
Those changes look to be in line with the renaming activity we started to move from IO Atala Prism to Hyperledger Identus organisation. I would leave the approval to the codeowner of Edge Agent SDK KMP though.
I have made a new commit and added a request to update the README file of the test before approval.

File: tests/end-to-end/src/test/resources/example.properties
Comment (mkbreuningIOHK) [inline tests/end-to-end/src/test/resources/example.properties:2]: As this is changed, the README.md file should be changed accordingly.

**Review by mkbreuningIOHK — CHANGES_REQUESTED** [review]

File: tests/end-to-end/build.gradle.kts
Comment (yshyn-iohk) [inline tests/end-to-end/build.gradle.kts:8]: It's better to keep these changes until the legal approves moving the repo under the Hyperledger org.
Otherwise, it might be difficult to release a new package: each group maintains the secrets that allow them to publish packages.

**Review by hamada147 — APPROVED** [review]

File: tests/end-to-end/build.gradle.kts
Comment (amagyar-iohk) [reply tests/end-to-end/build.gradle.kts:8]: this is the automation package, there's no artifact being released from here... but should I revert this change @yshyn-iohk ?

**Pull request merged**
